app/experimenter/experiments/tests/test_models.py

```diff
@@ -1115,6 +1115,106 @@ def test_monitoring_dashboard_url_returns_url_when_experiment_is_complete(self):
             ),
         )
 
+    def test_rollouts_monitoring_dashboard_url_is_none_when_rollout_not_begun(self):
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.CREATED,
+            slug="rollout-1-slug",
+            status=NimbusExperiment.Status.DRAFT,
+        )
+        self.assertIsNone(experiment.rollout_monitoring_dashboard_url)
+
+    def test_rollouts_monitoring_dashboard_url_is_none_on_first_day(self):
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.CREATED,
+            slug="rollout-1-slug",
+            status=NimbusExperiment.Status.DRAFT,
+        )
+
+        NimbusChangeLogFactory.create(
+            experiment=experiment,
+            old_status=NimbusExperiment.Status.DRAFT,
+            new_status=NimbusExperiment.Status.LIVE,
+            changed_on=datetime.date.today(),
+        )
+
+        self.assertIsNone(experiment.rollout_monitoring_dashboard_url)
+
+    def test_rollouts_monitoring_dashboard_returns_url_one_day_after_start(self):
+        yesterday = datetime.date.today() - datetime.timedelta(days=1)
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.CREATED,
+            slug="rollout-1-slug",
+            status=NimbusExperiment.Status.DRAFT,
+        )
+        NimbusChangeLogFactory.create(
+            experiment=experiment,
+            old_status=NimbusExperiment.Status.DRAFT,
+            new_status=NimbusExperiment.Status.LIVE,
+            changed_on=yesterday,
+        )
+
+        expected_slug = experiment.slug.replace("-", "_")
+        self.assertEqual(
+            experiment.rollout_monitoring_dashboard_url,
+            settings.ROLLOUT_MONITORING_URL.format(
+                slug=expected_slug,
+            ),
+        )
+
+    def test_rollouts_monitoring_dashboard_returns_correct_formatted_url(self):
+        yesterday = datetime.date.today() - datetime.timedelta(days=1)
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.CREATED,
+            slug="rollout-1-slug",
+            status=NimbusExperiment.Status.DRAFT,
+        )
+        NimbusChangeLogFactory.create(
+            experiment=experiment,
+            old_status=NimbusExperiment.Status.DRAFT,
+            new_status=NimbusExperiment.Status.LIVE,
+            changed_on=yesterday,
+        )
+
+        expected_slug = experiment.slug.replace("-", "_")
+        url = experiment.rollout_monitoring_dashboard_url
+        actual_slug = url[(url.index("::") + 2) :]  # take a slice of the url after '::'
+        self.assertEqual(expected_slug, actual_slug)
```

Comment:
I would just hard code the `expected_slug` here instead of transforming it since we're explicitly setting the original slug so we know what the transformed version is going to look like.  And it's fine to just check the whole URL instead of the slug separately, this kind of string querying is finnicky, if you wanted to really do this you'd want to use a regex, but I don't think it's necessary.